testdata/test-ssh.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I feel it is better to generate this key during testing. Otherwise some "secret leakage alert" tool may raise false-positive.

Reply:
ssh key is now generated during the test.